docs-ref-services/preview/core-https-readme.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Azure Core HTTP client library for JavaScript
+keywords: Azure, javascript, SDK, API, @azure/core-https, 
+author: maggiepint
+ms.author: magpint
```

Comment:
@xirzec We should get your name in here!

Reply:
out of curiosity, what does having an author tag in here actually do?

Reply:
Good question

@sima-zhu ?

Reply:
Have an issue to track this: https://github.com/Azure/azure-sdk-tools/issues/1416
Author is using by docs.ms
![image](https://user-images.githubusercontent.com/48036328/108279687-d6710780-7131-11eb-8773-088834a8a13c.png)